sunpy/physics/tests/test_differential_rotation.py

```diff
@@ -84,49 +85,9 @@ def seconds_per_day():
     return 24 * 60 * 60.0 * u.s
 
 
-def test_single(seconds_per_day):
-    rot = diff_rot(10 * seconds_per_day, 30 * u.deg)
-    assert_quantity_allclose(rot, 136.8216 * u.deg, rtol=1e-3)
-
-
-def test_array(seconds_per_day):
-    rot = diff_rot(10 * seconds_per_day, np.linspace(-70, 70, 2) * u.deg)
-    assert_quantity_allclose(rot, Longitude(np.array([110.2725, 110.2725]) * u.deg), rtol=1e-3)
-
-
-def test_synodic(seconds_per_day):
-    rot = diff_rot(10 * seconds_per_day, 30 * u.deg, rot_type='howard', frame_time='synodic')
-    assert_quantity_allclose(rot, 126.9656 * u.deg, rtol=1e-3)
-
-
-def test_sidereal(seconds_per_day):
-    rot = diff_rot(10 * seconds_per_day, 30 * u.deg, rot_type='howard', frame_time='sidereal')
-    assert_quantity_allclose(rot, 136.8216 * u.deg, rtol=1e-3)
-
-
-def test_howard(seconds_per_day):
-    rot = diff_rot(10 * seconds_per_day, 30 * u.deg, rot_type='howard')
-    assert_quantity_allclose(rot, 136.8216 * u.deg, rtol=1e-3)
-
-
-def test_allen(seconds_per_day):
-    rot = diff_rot(10 * seconds_per_day, 30 * u.deg, rot_type='allen')
-    assert_quantity_allclose(rot, 136.9 * u.deg, rtol=1e-3)
-
-
-def test_snodgrass(seconds_per_day):
-    rot = diff_rot(10 * seconds_per_day, 30 * u.deg, rot_type='snodgrass')
-    assert_quantity_allclose(rot, 135.4232 * u.deg, rtol=1e-3)
-
-
-def test_rigid(seconds_per_day):
-    rot = diff_rot(10 * seconds_per_day, [0, 30, 60] * u.deg, rot_type='rigid')
-    assert_quantity_allclose(rot, [141.844 * u.deg] * 3, rtol=1e-3)
-
-
-def test_fail(seconds_per_day):
-    with pytest.raises(ValueError):
-        diff_rot(10 * seconds_per_day, 30 * u.deg, rot_type='garbage')
+def test_diff_rot_deprecated_warning(seconds_per_day):
+    with pytest.warns(SunpyDeprecationWarning, match='The diff_rot function is deprecated and may be removed in version 6.1.\n        Use sunpy.sun.models.diff_rot instead.'):
```

Comment:
```suggestion
    with pytest.warns(SunpyDeprecationWarning, match='The diff_rot function is deprecated'):
```
Don't need to match the entire message.